packages/jest-environment/src/index.ts

```diff
@@ -172,6 +172,15 @@ export interface Jest {
    * local module state doesn't conflict between tests.
    */
   isolateModules(fn: () => void): Jest;
+  /**
+   * A flag that indicates whether the Jest environment has been torn down.
+   * @example
```

Comment:
Hm.. Not sure if 'flag' sounds is right here. Perhaps something like this:

```suggestion
   * Returns `true` if Jest environment has been torn down.
   *
   * @example
```

Reply:
done